homeassistant/components/switcher_kis/entity.py

```diff
@@ -0,0 +1,20 @@
+"""Base class for Switcher entities."""
+
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .coordinator import SwitcherDataUpdateCoordinator
+
+
+class SwitcherKisEntity(CoordinatorEntity[SwitcherDataUpdateCoordinator]):
```

Comment:
```suggestion
class SwitcherEntity(CoordinatorEntity[SwitcherDataUpdateCoordinator]):
```
Can we name this `SwitcherEntity` or `SwitcherBaseEntity`, the `KIS` (= Keep It Simple) was historically to separate this integration with another custom one `SwitcherAIO`